pkg/features/kube_features.go

```diff
@@ -1338,7 +1338,8 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 
 	genericfeatures.WatchList: {
 		{Version: version.MustParse("1.27"), Default: false, PreRelease: featuregate.Alpha},
-		{Version: version.MustParse("1.32"), Default: true, PreRelease: featuregate.Beta},
+		// switch this back to false because the json and proto streaming encoders appear to work better.
+		{Version: version.MustParse("1.32"), Default: false, PreRelease: featuregate.Beta},
```

Comment:
streaming collections were only in 1.33... are we going to backport toggling this default to false to 1.32, or leave it on in 1.32 and default it off in 1.33? I think this should match the actual state at 1.32 HEAD